platformio.ini

```diff
@@ -0,0 +1,24 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[platformio]
+src_dir = .
+
+[env]
+framework = arduino
+lib_deps =
+	bblanchon/ArduinoJson@^6.20.0
+build_flags = -D DEBUG_DELAY=4000
```

Comment:
Any particular reason to define this here instead of config.h?

Reply:
Hi!

Quite thanks for the PIO support and the warning fix!!

The main reason for DEBUG_DELAY in platformio.ini instead of config.h is ... :-/ ... forgotten quick & dirty hack.

PIO's serial monitor behave a little bit different than Arduino's... (or it depends on the boards lib).
In my last RP2040 based project ```while (!Serial);``` did not work at all. And because PIO's serial monitor doesn't reset the board once ready, you'll loose the first debug messages.
So I placed it into platformio.ini because I though that's mainly a developer option and not something for regular 'maker' configuration. Also because it highly depends on the used board, but also on your IDE's performance at all.

However. A fixed delay is shitty in any case!
I can test if ```while (!Serial);``` will work with my ESP32 and if so, we could do something like ```while (!Serial && millis() < DEBUG_DELAY_MAX);```

Reply:
Clicking "Upload and Monitor" in vscode without DEBUG_DELAY defined seems to work just like it did with arduino's serial monitor and still captures the initial log messages. Maybe it's just differences in our tooling or the boards we're using. But either way it's not going to break anything and if you need it it's likely other developers might too so I'm fine leaving it.